tutorials/navigation/navigation_using_navigationagents.rst

```diff
@@ -3,145 +3,177 @@
 Using NavigationAgents
 ======================
 
-NavigationsAgents are helper nodes to facilitate common calls to the NavigationServer API
-on behalf of the parent actor node in a more convenient manner for beginners.
+NavigationsAgents are helper nodes that combine functionality 
+for pathfinding, pathfollowing and agent avoidance for a Node2D/3D inheriting parent node.
```

Comment:
```suggestion
for pathfinding, path following and agent avoidance for a Node2D/3D inheriting parent node.
```
It irks me that pathfinding is common, but pathfollowing seems to be path following everywhere. 🤷 Anyway, if there are other mentions, those should be changed as well.